components/feature/sitepermissions/src/main/java/mozilla/components/feature/sitepermissions/SitePermissionsFeature.kt

```diff
@@ -761,6 +768,17 @@ class SitePermissionsFeature(
                     shouldSelectRememberChoice = true
                 )
             }
+            is ContentStorageAccess -> {
+                createContentStorageAccessPermissionPrompt(
+                    context,
+                    host,
+                    permissionRequest,
+                    R.string.mozac_feature_sitepermissions_storage_access_title,
+                    android.R.drawable.ic_dialog_alert,
```

Comment:
Nit: Maybe we could use a [cookie icon](https://design.firefox.com/icons/viewer/#cookie)?  :)